examples/librispeech/s0/README.md

```diff
@@ -0,0 +1,17 @@
+# Performance Record
+
+## Conformer Result
+
+* Feature info: using fbank feature, dither=0, cmvn, speed perturb
+* Training info: lr 0.004, batch size 12, 8 gpu, acc_grad 4, 70 epochs, dither 0.0
```

Comment:
please make the default parameter in train_conformer.yaml be consistent with the value here.